src/bug-filing/common/issue-filing-url-string-utils.ts

```diff
@@ -4,36 +4,35 @@ import { EnvironmentInfo } from '../../common/environment-info-provider';
 import { title } from '../../content/strings/application';
 import { CreateIssueDetailsTextData } from './../../common/types/create-issue-details-text-data';
 
-export class IssueFilingUrlStringUtils {
-    public static footer(environmentInfo: EnvironmentInfo): string {
+export type StringUtils = {
+    footer: (info: EnvironmentInfo) => string;
+    collapseConsecutiveSpaces: (input: string) => string;
+    markdownEscapeBlock: (input: string) => string;
+    getSelectorLastPart: (selector: string) => string;
+    standardizeTags: (data: CreateIssueDetailsTextData) => string[];
+};
+
+export const IssueFilingUrlStringUtils: StringUtils = {
```

Comment:
Do we really need a object with a type that just contains 5 functions? I was thinking that these 5 functions as separate functions in `issue-filling-url-string-utils.ts` file should be good enough. Thoughts?

Reply:
I see your point. The functions are stand alone from the execution point of view (i.e. they don't share any kind of state, that's why they were static functions). But, from the usage point of view, we're using this functions pretty close together. If you look at `src\bug-filing\github\create-github-bug-filing-url.ts` you'll see I'm creating a function by passing an instance of `StringUtils`, which looks more convenient than passing 5 separated (but usage-related functions).